libavogadro/src/extensions/crystallography/crystallographyextension.cpp

```diff
@@ -2191,6 +2195,13 @@ namespace Avogadro
     CE_CACTION_DEBUG(PrimitiveReduceIndex);
     CE_CACTION_ASSERT(PrimitiveReduceIndex);
 
+    // PrimitiveReduceStandardIndex,
+    a = new QAction(tr("Reduce Cell (&Primitive and Standardize)"), this);
```

Comment:
The rest of the patch looks fine - can we pick a better name, like "Reduce and Standardize"?